src/states/App.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useState} from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import {Button, Container} from "@material-ui/core";
+import MyCard from "../components/MyCard/MyCard";
+import Header from "../components/Header/Header";
+import MyModal from "./MyModal/MyModal";
+import arrayCards from '../data/arrayCards.json'
+import ICardsDataDTO from "../models/ICardsDataDTO";
+
+const useStyles = makeStyles({
+    cardContainer: {
+        display: "flex",
+        flexWrap: "wrap"
+    },
+    addCardBtn: {
+        margin: "1rem auto",
+        display: "flex",
+        justifyContent: "center"
+    }
+});
+
+const App = () => {
+    const [cards] = useState<ICardsDataDTO[]>(arrayCards)
+    const [cardHeaderText, setCardHeaderText] = useState<string>('')
+    const [cardBodyText, setCardBodyText] = useState<string>('')
+    const [isOpen, setIsOpen] = useState<boolean>(false);
+    const classes = useStyles();
+
+    const handleOpen = () => {
```

Comment:
You don't need this 3 methods here. You can replace it with only one using `prevState` or even not create methods and do it inline like this: `() => setIsOpen(true)`.